### PR TITLE
Replace removed deprecated methods in PS9

### PR DESCRIPTION
**Pull request by 0x346e3730**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Replace removed deprecated methods in PS9 that were used in the 
| Type?         | bug fix
| BC breaks?    | yes
| Deprecations? | yes
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/32180.
| How to test?  | See ticket.

### CODE REVIEW

File: controllers/admin/AdminPsThemeCustoConfiguration.php [hunk 1]
```diff
@@ -502,19 +502,19 @@ public function setFinalList($aList)
                 if ($sType == 'pages') {
                     foreach ($aElementsList as $sController => $aPage) {
                         $aModuleFinalList[$sSegmentName][$sType][$sController]['name'] = $sController;
-                        $aModuleFinalList[$sSegmentName][$sType][$sController]['displayName'] = $this->l($aPage[0]);
+                        $aModuleFinalList[$sSegmentName][$sType][$sController]['displayName'] = $this->trans($aPage[0]);
```
Comment (mflasquin) [inline controllers/admin/AdminPsThemeCustoConfiguration.php:505]: Now, trans are not escaped by default. Is there a risk that $aPage[0] contains user entry ? If yes, I think you can use htmlspechialchars on the parameter.

**Review by mflasquin — APPROVED** [review]

File: controllers/admin/AdminPsThemeCustoConfiguration.php (hunk 1 above)
Comment (0x346e3730) [reply controllers/admin/AdminPsThemeCustoConfiguration.php:505]: I'm not sure tbh I'm investing

Comment (0x346e3730) [reply controllers/admin/AdminPsThemeCustoConfiguration.php:505]: To me it's not user's input

**Review by boherm — APPROVED** [review]

**Review by MhiriFaten — APPROVED** [review]
Hello @0x346e3730 ,

I have checked your PR,
the Advanced Customization tab is well displayed and the error is no more displayed!

image

It is QA approved :heavy_check_mark: 

Thank you.

**Comment by lartist** [comment]
Thanks @0x346e3730

**Pull request merged**